atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -87,7 +113,7 @@ def run(self):
             if not build:
                 break
```

Comment:
> which would be adjusted to 21

Not sure if this would actually happen. If `getNextRelease` returns `21.2` and that build doesn't exist, this `break` statement will execute - meaning that `self.increment_release` is not invoked, thus release value won't be adjusted.